cli-commands/commands.js

```diff
@@ -330,16 +330,14 @@ const commands = [
 	},
 	{
 		command: 'opt-out',
-		description: 'Disables analytics',
+		description: 'Disables user tracking analytics.',
```

Comment:
Even worse :D

"Opt out of the event tracking etherlime uses in order to improve itself (please dont)" would be much better